search/utils/constants.py

```diff
@@ -1,3 +1,5 @@
+# These should really belong in settings.py
```

Comment:
@shriyanka Okies. I guess this is debatable. Depends on context of the situation.